terraform/yaml/eks/Pulumi.yaml

```diff
@@ -0,0 +1,6 @@
+name: eks
+runtime:
+  name: yaml
+  options:
+    compiler: ./run.sh
+description: Terrayamlulmi
```

Comment:
`Terrayamlumi`?

Reply:
Terraform => yaml => pulumi.

Terrible and contrived just like the PR. =)